recipes/rpmcmc/meta.yaml

```diff
@@ -0,0 +1,31 @@
+{% set version = "0.2" %}
+{% set sha256 = "721eb4d3202c925fb3f564902e1e753634c56e4871e78f652401d77c26532b2c" %}
+
+package:
+  name: rpmcmc
+  version: {{ version }}
+
+source:
+  url: http://daweb.ism.ac.jp/yoshidalab/motif/rpmcmc-{{ version }}.tar.gz 
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True # [osx]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+
+test:
+  commands:
+    - multi_motif_sampler 2>&1 | grep "usage"
+
+about:
+  home: http://daweb.ism.ac.jp/yoshidalab/motif/
+  license: Other
```

Comment:
Does the license allow redistribution of the software?

Reply:
There is no license...

Reply:
So @bgruening , is a license required for inclusion in bioconda?

Reply:
Yes, no license is legally very complicated. We need to be sure that we are allowed to distribute the software.

Reply:
I understand that. I'll close the PR for now. Would an OK from the author be sufficient? Or would we really need something in writing (ie. a license :))?

Reply:
Leave the PR open and point the author to this issue. This is probably just unintentional. A lot of Bioinformatic software has no license in the beginning, it's nothing that we are trained at.

The License file would be super, but if this is strictly needed depends on the license that will be chosen. GPL, for example, requires a license file.